hybrid-standalone/modules/ROOT/pages/managing-properties-hybrid.adoc

```diff
@@ -0,0 +1,117 @@
+= Managing Properties
+
+Manage application properties for your Mule applications deployed to hybrid standalone instances through centralized configuration management, enabling dynamic property updates, environment-specific settings, and secure property handling across your infrastructure.
+
+== Overview
+
+Runtime Manager provides comprehensive property management capabilities for applications deployed to hybrid standalone instances, enabling you to:
+
+* Define and manage application properties centrally through Runtime Manager
+* Override application-bundled properties without redeployment
+* Configure environment-specific settings for different deployment stages
+* Secure sensitive configuration data through encrypted property storage
+* Manage properties across individual servers, server groups, and clusters
+
+**Property Management Approaches**
+
+* **Properties Files**: Include property placeholder YAML files in application deployment archives
```

Comment:
We should use bold only  for UI elements. You can either

- Use a definitions list:
_Properties Files:: Include property placeholder YAML files in application deployment archives._
- Or remove the bold internal headings:
_* Include property placeholder YAML files in application deployment archives._